src/Chullo.php

```diff
@@ -85,6 +86,31 @@ public function getResource($uri = "",
     }
 
     /**
+     * Gets a Fedora resoure's headers.
+     *
+     * @param string    $uri            Resource URI
+     * @param string    $transaction    Transaction id
+     *
+     * @return array    Headers of a resource.
+     */
+    public function getResourceHeaders($uri = "",
+                                       $transaction = "") {
+
+        // Ensure uri takes transaction into account.
+        $uri = $this->prepareUri($uri, $transaction);
+
+        // Send the request.
+        $response = $this->client->request(
+            'GET',
```

Comment:
Getting resource headers is implemented with the HEAD http method.

https://wiki.duraspace.org/display/FEDORA40/RESTful+HTTP+API+-+Containers#RESTfulHTTPAPI-Containers-GreenHEADtrueRetrievetheresourceheaders